src/express-server/routes/products-routes.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import * as products from "../controllers/products-controllers";
+
+export const productsRouter = express.Router();
+
+productsRouter.route("/api/products")
+  .get((req, res) => {
```

Comment:
1. Please don't use filenames with dash. If you want you may call it like `products.routes.ts`, or, as it's situated in `routes` folder already it may be simply called `products.ts`
2. Extract controllers to separate files